test/cppsuite/test_harness/workload/database_operation.h

```diff
@@ -161,18 +154,12 @@ class database_operation {
         WT_DECL_RET;
         wt_timestamp_t ts;
         std::vector<WT_CURSOR *> cursors;
-        std::vector<std::string> collection_names;
+        std::vector<std::string> collection_names = std::move(tc->database.get_collection_names());
```

Comment:
Shouldnt it get the collections in the loop ?
What if a collection is created/deleted during the test ?

Reply:
As discussed, this is a more complicated use case that we'll look at addressing in the future. At the moment, we're just providing a way for other components to access collection names.